File: Atividade73/model/Conta.js
class Conta{
    #numero
    #agencia
    #saldo

    get numero(){
        return this.#numero
    }
    get agencia(){
        return this.#agencia
    }
    get saldo(){
        return this.#saldo
    }
    set numero(numero){
        this.#numero=numero
    }
    set agencia(agencia){
        this.#agencia=agencia
    }
    set saldo(saldo){
        this.#saldo=saldo
    }
    
}
export default Conta

File: Atividade21/Conta.java

public class Conta {
    private double saldo;
    private int codCliente;

    public void setCodCliente(int codCliente) {
        this.codCliente = codCliente;
    }
    public void setSaldo(double saldo) {
        this.saldo = saldo;
    }
    public int getCodCliente() {
        return codCliente;
    }
    public double getSaldo() {
        return saldo;
    }
    public void transferencia(double valor){
        this.saldo -=valor;
    }
}
